src/test/ui/transmute/transmute-padding-ice.rs

```diff
@@ -0,0 +1,29 @@
+#![crate_type = "lib"]
```

Comment:
FWIW it is useful to name regression tests after the issues they come from. This has couple purposes:

* It is easy to see this is a regression test;
* Resolving to the original issue report takes a single dereference (rather than having to go through `git log`, looking at the commit message and only then finding the issue number to load up.)

Not a huge deal either way though.